### ISSUE TITLE
example/data_staging/io_staging_dict.py - got an unexpected keyword argument 'name'

### ISSUE THREAD

**Issue by FranklinBetten** [radical-cybertools/radical.pilot#1518]
in radical pilot examples/data_staging/

python io_staging_dict.py gives following error 
```
Traceback (most recent call last):
  File "io_staging_dict.py", line 50, in <module>
    session = rp.Session(name=session_name)
TypeError: __init__() got an unexpected keyword argument 'name'
```

**Comment by FranklinBetten** [comment]
after replacing name with uid there is a new error 

Traceback (most recent call last):
  File "io_staging_dict.py", line 102, in <module>
    umgr = rp.UnitManager(session, rp.SCHED_DIRECT_SUBMISSION)
AttributeError: 'module' object has no attribute 'SCHED_DIRECT_SUBMISSION'


To solve this we removed the rp.SHED_DIRECT_SUBMISSION option from the rp.UnitManager call. 

        # Combine the ComputePilot, the ComputeUnits and a scheduler via
        # a UnitManager object.
        #umgr = rp.UnitManager(session, rp.SCHED_DIRECT_SUBMISSION)
        umgr = rp.UnitManager(session)


After that correction there was a new issue that showed up ... 

wait for 1 unit(s)
        O[Callback]: ComputeUnit 'unit.000000' state: UMGR_SCHEDULING_PENDING.
[Callback]: ComputeUnit 'unit.000000' state: UMGR_SCHEDULING.
[Callback]: ComputeUnit 'unit.000000' state: UMGR_STAGING_INPUT_PENDING.
[Callback]: ComputeUnit 'unit.000000' state: UMGR_STAGING_INPUT.
/[Callback]: ComputeUnit 'unit.000000' state: AGENT_STAGING_INPUT_PENDING.
\[Callback]: ComputePilot 'pilot.0000' state: PMGR_ACTIVE_PENDING.
\[Callback]: ComputeUnit 'unit.000000' state: FAILED.
-                                                                     ok
caught Exception: 'ComputeUnit' object has no attribute 'execution_locations'
closing session
[Callback]: ComputePilot 'pilot.0000' state: PMGR_ACTIVE.
close unit manager                                                            ok
close pilot manager                                                            \
wait for 1 pilot(s)
                                                                         timeout
                                                                              ok
session lifetime: 21.4s                                                       ok
Traceback (most recent call last):
  File "io_staging_dict.py", line 123, in <module>
    (unit.uid, unit.execution_locations, unit.state,
AttributeError: 'ComputeUnit' object has no attribute 'execution_locations'

**Comment by vivek-bala** [comment]
Hey Franklin, do you have any other traceback from the error? Can you run it with RADICAL_PILOT_VERBOSE=DEBUG please.

**Comment by andre-merzky** [comment]
No need for the trace - I'm afraid that the example (and some others outside of the canonical set) are badly out of sync, and fell behind some recent (and not so recent) API changes.  That attribute (`execution_locations`) is not supported anymore (it was never correctly implemented anyway IIRC).

**Comment by andre-merzky** [comment]
Franklin, would you be able to take this ticket on to update the data staging examples (or to remove them where they are redundant with the canonical examples)?

**Comment by andre-merzky** [comment]
fixed via #1519
